milk-planning/src/test/scala/dev/atedeg/mdm/milkplanning/types/ActionsTest.scala

```diff
@@ -0,0 +1,71 @@
+package dev.atedeg.mdm.milkplanning.types
+
+import java.time.LocalDateTime
+
+import cats.data.{ NonEmptyList, NonEmptyMap, Writer }
+import cats.implicits.catsKernelOrderingForOrder
+import cats.syntax.all.*
+import org.scalatest.GivenWhenThen
+import org.scalatest.featurespec.AnyFeatureSpec
+import org.scalatest.matchers.should.Matchers
+
+import dev.atedeg.mdm.milkplanning.*
+import dev.atedeg.mdm.milkplanning.OutgoingEvent.OrderMilk
+import dev.atedeg.mdm.milkplanning.utils.QuintalsOfMilkOps.*
+import dev.atedeg.mdm.milkplanning.utils.QuintalsOfMilkOps.given
+import dev.atedeg.mdm.products.{ CheeseType, Grams, Product }
+import dev.atedeg.mdm.products.Product.*
+import dev.atedeg.mdm.utils.*
+import dev.atedeg.mdm.utils.given
+import dev.atedeg.mdm.utils.monads.*
+
+trait Fixture {
```

Comment:
@ndido98 called this trait `Mocks` in his PR, we should stick with one name, personally I prefer `Mocks`